### PR TITLE
Remove KWIML third party code

### PR DESCRIPTION
**Pull request by seanm**
Seems like it was only needed pre-C++11.

### CODE REVIEW

File: Modules/Bridge/VtkGlue/itk-module-init.cmake [hunk 1]
```diff
@@ -52,7 +52,7 @@ set(_required_vtk_libraries
   )
 if(ITK_WRAP_PYTHON)
   list(APPEND _required_vtk_libraries ${_target_prefix}WrappingPythonCore
-    ${_target_prefix}CommonCore ${_target_prefix}CommonDataModel ${_target_prefix}kwiml ${_target_prefix}CommonExecutionModel)
+    ${_target_prefix}CommonCore ${_target_prefix}CommonDataModel ${_target_prefix}CommonExecutionModel)
```
Comment (dzenanz) [inline Modules/Bridge/VtkGlue/itk-module-init.cmake:55]: This commit should be broken into two. First one to remove the remaining references to KWIML, the second one to remove KWIML itself.

**Review by dzenanz — APPROVED** [review]

File: Modules/Bridge/VtkGlue/itk-module-init.cmake (hunk 1 above)
Comment (seanm) [reply Modules/Bridge/VtkGlue/itk-module-init.cmake:55]: OK. But why?

Comment (dzenanz) [reply Modules/Bridge/VtkGlue/itk-module-init.cmake:55]: Those are two logical changes, and it is preferable that each change has its own commit. In this case it is easy to separate them, but sometimes it is not. In spirit, this is similar to Third party updates: one commit to change the branch name or hash, one commit which adds the actual third party changes which can be big and usually crowd out all other changes.

Comment (seanm) [reply Modules/Bridge/VtkGlue/itk-module-init.cmake:55]: Done.

**Review by dzenanz — APPROVED** [review]

**Review by thewtex — APPROVED** [review]
Great!

**Pull request merged**